tools/ci/flink-ci-tools/src/main/java/org/apache/flink/tools/ci/utils/shade/ShadeParser.java

```diff
@@ -46,7 +46,6 @@ public final class ShadeParser {
                             + "(?<groupId>.*?):"
                             + "(?<artifactId>.*?):"
                             + "(?<type>.*?):"
-                            + "(?:(?<classifier>.*?):)?"
```

Comment:
Had to roll-back this addition to keep the behavior in sync with the NoticeParser, which currently doesn't account for classifiers and just parses them as part of the version.

Will be fixed in a follow-up.

Reply:
Sure, okay. Baby steps.